src/magic_folder/config.py

```diff
@@ -232,13 +233,20 @@
             [upload_duration_ns]  INTEGER        -- nanoseconds the last upload took
         )
         """,
+        """
+        --- This table represents our notion of conflicts (although they are also represented
+        --- on disk, our representation is canonical as the filesystem is part of the API)
+        CREATE TABLE [conflicted_files]
+        (
+            [name]             TEXT NOT NULL,    -- mangled name in UTF-8
+            [conflict_author]  TEXT NOT NULL     -- another participant who conflicts
```

Comment:
I think we also want to store the snapshot cap we wrote to the conflict file.

We may also want to record the pathstate of the conflict file here (particularly if we don't treat `.conflict-*` files as snapshots, see https://github.com/LeastAuthority/magic-folder/pull/539#discussion_r685525375 for discussion).

Reply:
Yes, we may indeed -- but only when we provide a way to resolve conflicts (then we'll need the Snapshot capability to put in our parents list for the resolution snapshot).

Reply:
We want to at least store the snapshot cap now, since it isn't recoverable in the future, when we *can* resolve conflicts.

Reply:
Basically, if don't store the snapshot cap now, we won't have enough information to resolve the conflict in the future.